orttraining/orttraining/python/training/experimental/exporter.py

```diff
@@ -0,0 +1,20 @@
+import torch
+import torch.onnx.symbolic_helper
+import torch.onnx.utils
+
+# Only this version is tested.
+onnx_version = 13
+
+
+def _export_jit_graph_to_onnx_model_proto(graph, operator_export_type):
```

Comment:
If you search for this string in this PR, you will find
```c++
static std::string ExportToOnnx(
    std::shared_ptr<torch::jit::Graph> graph,
    const at::ArrayRef<c10::IValue>& args) {
#ifdef USE_CUDA
  NvtxRange range(__func__);
#endif
  // ONNX exporter modifies the graph in-place, so we
  // need to clone it to avoid interaction between
  // Pytorch's JIT mechanism and ONNX graph.
  std::shared_ptr<torch::jit::Graph> new_subgraph(graph->copyUnique().release());
  // Acquire GIL since Python is not multi-threading.
  pybind11::gil_scoped_acquire guard{};
  // Retrieve Python exporter function.
  pybind11::function export_to_onnx =
      pybind11::reinterpret_borrow<pybind11::function>(
          pybind11::module::import("onnxruntime.training.experimental.exporter")
              .attr("_export_jit_graph_to_onnx_model_proto"));
  // Fill types up. The sub-graphp from LazyTensor doesn't
  // contain input shapes.
  SetArgTypes(args, new_subgraph);
  // Execute Python function.
  auto result = export_to_onnx(new_subgraph, ::torch::onnx::OperatorExportTypes::ONNX);
  return result.cast<std::string>();
}
```